src/pages/Landing/index.tsx

```diff
@@ -54,24 +55,27 @@ const TitleText = styled.h1<{ isDarkMode: boolean }>`
 
   @media screen and (min-width: ${BREAKPOINTS.sm}px) {
     font-size: 48px;
+    line-height: 56px;
   }
 
   @media screen and (min-width: ${BREAKPOINTS.md}px) {
-    font-size: 72px;
+    font-size: 64px;
+    line-height: 72px;
   }
 `
 
 const SubText = styled.h3`
   color: ${({ theme }) => theme.textSecondary};
   font-size: 16px;
   line-height: 24px;
-  font-weight: 400;
+  font-weight: 500;
```

Comment:
i took a peek at the figma and I couldn't find where font-weight was specified to be 500, seems to be 400 in most places? 

If so, this Subtext has the same text specs as ThemedText.BodySecondary so maybe we could use that component instead and apply the extra styles to that?

Reply:
the figma keeps changing sadly, so you can ignore that.

and it's not the same as bodysecondary because there are weird colors and stuff. it also kept changing so wanted to avoid using the style. after launch i'm gonna come back to refactor this usage, agreed